bob-common/src/metrics/mod.rs

```diff
@@ -390,6 +392,17 @@ fn build_prometheus(node_config: &NodeConfig) -> PrometheusRecorder {
         }
     };
     tokio::spawn(future);
+
+    let handle = recorder.handle();
```

Comment:
Please, add comment, why this logic is needed (also add a full url of associated issue: https://github.com/qoollo/bob/issues/788)